File: packages/expo-p256/ios/P256Options.swift
import ExpoModulesCore

internal struct P256Options: Record {
  @Field
  var authenticationPrompt: String?

  @Field
  var keychainAccessible: P256Accessible = .whenUnlocked

  @Field
  var keychainService: String?

  @Field
  var requireAuthentication: Bool
}

File: packages/expo-p256/android/src/main/java/expo/modules/securestore/P256Options.kt
package expo.modules.p256

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record
import java.io.Serializable

class P256Options(
  // Prompt can't be an empty string
  @Field var authenticationPrompt: String = "",
  @Field var keychainService: String = P256Module.DEFAULT_KEYSTORE_ALIAS,
  @Field var requireAuthentication: Boolean = false,
) : Record, Serializable
